runtime/hashing/common/test/aws/smithy/kotlin/runtime/hashing/Crc32cTest.kt

```diff
@@ -60,4 +60,24 @@ class Crc32cTest {
         crc.digest()
         assertEquals(0U, crc.digestValue()) // checksum should be reset
     }
+
+    @Test
+    fun testNonAsciiInput() {
+        val crc = Crc32c()
+        val input = byteArrayOf(0)
+        crc.update(input, 0, 1)
+        val bytes = crc.digest()
+
+        assertEquals("Un1TUQ==", bytes.encodeBase64String())
+    }
+
+    @Test
+    fun testLargeInput() {
+        val crc = Crc32c()
+        val input = ByteArray(1024) { 0 }
+        crc.update(input, 0, input.size)
+
+        val bytes = crc.digest()
+        assertEquals("7q7efA==", bytes.encodeBase64String())
+    }
```

Comment:
**Question**: How were these new test values derived? Is there a wider suite of known-correct test cases from which we could draw?

Reply:
These were derived with [the crc32c Python package]( https://pypi.org/project/crc32c/). I took a look and it seems there's no widely-accepted suite of test cases. I found some in [RFC 3720](https://www.rfc-editor.org/rfc/rfc3720#appendix-B.4) but overall it's not focused on CRC, just in the appendix.